pkg/k8s/apis/cilium.io/client/crds/v2/ciliumclusterwidenetworkpolicies.yaml

```diff
@@ -1488,6 +1488,31 @@ spec:
                             type: object
                         type: object
                       type: array
+                    icmp:
```

Comment:
I wonder what's the "to" means.
Since port number is different for destination and source, we need "to".
However ICMP type is not, so I don't think "to" is necessary.

Reply:
Oh, you're totally right! We probably still need the plural though, even if `icmps` is a bit ugly...

Reply:
How about `ICMPs`?

Reply:
I think the first letter needs to be lower case.

Reply:
OK, so let's change to `icmps`.

Reply:
We can always change latter (until the v1.11 release) based on feedback from others :-)